lib/chef/provider/package/chocolatey.rb

```diff
@@ -136,14 +136,27 @@ def check_resource_semantics!
         #
         # @return [String] full path of choco.exe
         def choco_exe
-          @choco_exe ||=
+          @choco_exe ||= begin
+            path = choco_install_path
+
+            # GetEnvironmentVariable returns "" on failure.
+            if path.to_s.empty?
+              raise Chef::Exceptions::MissingLibrary.new("Could not find Chocolatey. Is it installed?")
```

Comment:
How about adding some info about this powershell command we run to find it, in case the user actually has chocolatey but there is something else that's broken?